packages/scripts/scripts/test-e2e.js

```diff
@@ -50,4 +50,10 @@ if ( hasCliArg( '--puppeteer-interactive' ) ) {
 	process.env.PUPPETEER_SLOWMO = getCliArg( '--puppeteer-slowmo' ) || 80;
 }
 
+if ( hasCliArg( '--puppeteer-local' ) ) {
```

Comment:
Hello @gziolo :)
I am not sure of 2 things here:
1. whether `local` is the best arg name
2. if checking arg with `--puppeteer-` as in example above is easy to reason about

Reply:
Actually, the defaults are:

```js
process.env.WP_BASE_URL = 'http://localhost:8889';
process.env.WP_USERNAME = 'admin';
process.env.WP_PASSWORD = 'password';
```
See: https://github.com/WordPress/gutenberg/blob/master/packages/e2e-test-utils/src/shared/config.js#L9

So there is no need to set them explicitly. We need the opposite, to have a way to override all those 3 values:



```js
if ( hasCliArg( '--wordpress-host' ) ) {
    process.env.WP_BASE_URL = getCliArg( '--wordpress-host' );
}
if ( hasCliArg( '--wordpress-username' ) ) {
    process.env. WP_USERNAME = getCliArg( '--wordpress-username' );
}
if ( hasCliArg( '--wordpress-password' ) ) {
    process.env. WP_PASSWORD = getCliArg( '--wordpress-password' );
}
```
or even better something which uses look up table and loops over it.